### PR TITLE
[HOPSWORKS-1287] Support secrets in ParameterStore

### PR DESCRIPTION
**Pull request by steffengr**
Add support for storing secrets in the AWS parameter store in
addition to the secrets manager.

### CODE REVIEW

File: hops/util.py [hunk 1]
```diff
@@ -167,8 +169,25 @@ def _parse_rest_error(response_dict):
         user_msg = response_dict[constants.REST_CONFIG.JSON_USR_MSG]
     return error_code, error_msg, user_msg
 
+def get_secret(project_name, secrets_store, secret_key):
```
Comment (Kim-Hammar) [inline hops/util.py:172]: Is it a public method? If so add docstring. If not, prepend "_" to the methodname.

File: hops/util.py [hunk 2]
```diff
@@ -204,6 +212,10 @@ def assumed_role():
     get_secret_value_response = client.get_secret_value(SecretId=secret_name)
     return json.loads(get_secret_value_response['SecretString'])[secret_key]
 
+def query_paramter_store(project_name, secret_key):
```
Comment (Kim-Hammar) [inline hops/util.py]: Is it a public method? If so add docstring. If not, prepend "_" to the methodname.

Comment (Kim-Hammar) [reply hops/util.py]: Also, there is a typo in the method name: `paramter`.

**Review by Kim-Hammar — CHANGES_REQUESTED** [review]

File: hops/util.py (hunk 1 above)
Comment (steffengr) [reply hops/util.py:172]: It's public. I added a doc string.

File: hops/util.py (hunk 2 above)
Comment (steffengr) [reply hops/util.py]: Shouldn't be public anymore. I fixed that and the typo.

**Review by Kim-Hammar — APPROVED** [review]

**Review by Kim-Hammar — APPROVED** [review]

**Pull request merged**